Source/WebCore/page/SettingsBase.cpp

```diff
@@ -437,6 +441,19 @@ void SettingsBase::layerBasedSVGEngineEnabledChanged()
 
 #endif
 
+#if ENABLE(MEDIA_SESSION)
+void SettingsBase::mediaSessionEnabledChanged()
+{
+#if USE(GLIB)
+    bool enabled = false;
+    if (m_page)
+        enabled = m_page->settings().mediaSessionEnabled();
+    auto& sessionManager = reinterpret_cast<MediaSessionManagerGLib&>(PlatformMediaSessionManager::sharedManager());
+    sessionManager.setDBusNotificationsEnabled(enabled);
```

Comment:
Good catch. We could either (a) make MediaSessionManager non-global, or (b) just don't add the setting?

We probably don't really _need_ the setting, do we? We've agreed it's not fully baked yet and shouldn't be exposed to users yet. Developers can enable the feature manually if desired?

Reply:
https://github.com/WebKit/WebKit/pull/8515

Reply:
I'm a bit late here I guess but the setting would have been helpful for Tangram since tabs don't share `WebKitSettings`, a user could enable it for a specific tab.

Reply:
We could have a websetting for media-session, but for MPRIS it would not be so easy due to the current singleton approach, I'm afraid.